src/components/PopUpAnimatedContainer/PopUpAnimatedContainer.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react'
 import { useSpring, animated } from 'react-spring'
 
+/**
+ * Component which can be used to display components in an animated popup container.
+ *
+ * @function
+ *
+ * @param {boolean} small - Parameter that is used to set the size of the popup.
+ * @param {boolean} large - Parameter that is used to set the size of the popup.
+ * @param {object} children - Parameter containing props used within the popup.
```

Comment:
I've changed the parameter description.

Reply:
The description 'Parameter containing props used within the popup' is kind of redundant because this is the universal description of a passed prop. Could you change this to:
`@param {object} children - Can contain child components`